tests/lib/pkgdb.sh

```diff
@@ -494,6 +494,25 @@ distro_get_package_extension() {
     esac
 }
 
+distro_get_installed_packages() {
```

Comment:
nitpick: since this function is not returning the list of installed packages, I'd call it `distro_print_installed_packages()`

Reply:
sure, makes sense, thanks